### ISSUE TITLE
[quest] [Zuldrak] [Betrayal]

### ISSUE THREAD

**Issue by MichaelCrow** [Stormforge-gg/Bugtracker-Fordring#1827]
[//]: # (Enclose links to things related to the bug using http://wowhead.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Zone] A Vision of the Past)
[//]: # ([NPC] Thoralius the Wise)
[//]: # ([Spell][Class] Frostfire Bolt)
[//]: # ([Talent][Class] Brain Freeze)
[//]: # ([Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([Npc][Drop] Thoralius the Wise)
[//]: # ([Web] Armory doesnt work)

**Description**:  During this quest, you are supposed to don a suit from Ensorcelled Choker and talk to Overlord Drakuru, then enslave a Blightblood troll with the Scepter of Domination to fight him.

So you do, however he periodically throws out blight crystals that afflict those that stand in the puddle it creates with a debuff thats not dispellable called Blight Fog, which deals like 2000 damage every 3 seconds. The trick is to not stand in it during the fight.

**Current behaviour**:  The problem is, once you do, which you have to as it lands on your location immediately afflicting you with it, its a permanent duration and doesnt drop once you walk out of the puddle, it seems to be an aura rather than an area periodic aura.

**Expected behaviour**:  Expected to be an area periodic aura so you dont auto die after 5 or 6 ticks.

**Comment by Cold-Martijn** [comment]
Duplicate of https://github.com/Stormforge-gg/Bugtracker-Fordring/issues/1758
